exception_handler.c

```diff
@@ -105,10 +105,7 @@ void the_exception (void)
   asm ("movia r22, process_stack_pointer)");
   asm ("stw sp, 0(r22)");
 
-  //move in address of ksp var into register
-  asm ("movia r23, ksp"); //compiles if i dont do &ksp, but ksp instead
-  //store sp register value into kernel stack address, which needs to be in a register
-  asm ("ldw sp, 0(r23)");
+  asm ("movia sp, ksp");  //move in address of ksp var into register sp
```

Comment:
This change was because I noticed that the value in register sp was often 0 when entering the interrupt_handler.